client/src/ui/molecules/document-survey/surveys.ts

```diff
@@ -1,18 +1,17 @@
-import { Doc } from "../../../../../libs/types/document";
 import { survey_duration, survey_rates } from "../../../env";
 
 export interface Survey {
   key: SurveyKey;
   bucket: SurveyBucket;
-  show: (doc: Doc) => boolean;
+  show: (doc: { mdn_url: string }) => boolean;
```

Comment:
Semantically better:

```suggestion
  show: (doc: Partial<Doc, "mdn_url">) => boolean;
```